model/setup.sql

```diff
@@ -0,0 +1,105 @@
+CREATE SCHEMA tournamentbuzz;
+USE tournamentbuzz;
+
+CREATE TABLE users (
+  email VARCHAR(255) NOT NULL UNIQUE CHECK(email REGEXP '^[A-Za-z0-9][A-Za-z0-9]*@[A-Za-z0-9][A-Za-z0-9]*\\.[A-Za-z0-9]*$'),
+  passw VARCHAR(255) NOT NULL CHECK(passw LIKE '%________%'),
+  uname VARCHAR(60),
+  sadmin BOOL DEFAULT FALSE NOT NULL, 
+  PRIMARY KEY(email)
+);
+
+CREATE TABLE tournaments (
+	id INT(10) NOT NULL UNIQUE AUTO_INCREMENT,
+    creator VARCHAR(255) NOT NULL,
+    sponsor VARCHAR(255) DEFAULT NULL,
```

Comment:
Is storing the sponsor necessary? Wouldn't executive/administrative information be stored in the description/stored by the user themselves?

Reply:
I'm keeping it for now